README.md

```diff
@@ -1,164 +1,17 @@
-# {{gear_name}} ({{gear_label}})
-
-## Overview
-
-*{Link To Usage}*
-
-*{Link To FAQ}*
+# fw-pipeline-io
 
 ### Summary
```

Comment:
Probably not necessary if as you said this repo is largely meant to remain private for internal FW use for the near future, but it would ultimately be helpful to explicitly list what needs to be installed in order to use the package.

Reply:
added requirements info